atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -267,35 +227,17 @@ def run_in_orchestrator(self):
         else:
             if pullspecs:
                 replacement_pullspecs = self._get_replacement_pullspecs(pullspecs)
```

Comment:
this variable should be set to {} or None before IF

Reply:
updated.